include/RAJA/policy/openmp/reduce.hpp

```diff
@@ -568,499 +235,49 @@ class ReduceMaxLoc<omp_reduce, T>
- *
- ******************************************************************************
- */
 template <typename T>
-class ReduceMaxLoc<omp_reduce_ordered, T>
+class ReduceMinLoc<omp_reduce_ordered, T>
+    : public detail::BaseReduceMinLoc<T, detail::ReduceOMPOrdered>
 {
-  using Reduce = RAJA::reduce::maxloc<T, Index_type>;
-
-  RAJA_INLINE T* asT(int i) const {
-    return reinterpret_cast<T*>(m_blockdata + i * s_block_offset);
-  }
-
-  RAJA_INLINE Index_type* asIndex(int i) const {
-    return reinterpret_cast<Index_type*>(m_idxdata + i * s_idx_offset);
-  }
-
 public:
-
-  ReduceMaxLoc() = delete;
-
-  //! Constructor takes default value (default ctor is disabled).
-  explicit ReduceMaxLoc(T init_val, Index_type init_loc)
-    : m_myID(getCPUReductionId()),
-      m_blockdata(getCPUReductionMemBlock(m_myID)),
-      m_idxdata(getCPUReductionLocBlock(m_myID)),
-      m_reduced_val(init_val),
-      m_reduced_idx(init_loc),
-      m_is_copy(false)
-  {
-    int nthreads = omp_get_max_threads();
-#pragma omp parallel for schedule(static, 1)
-    for (int i = 0; i < nthreads; ++i) {
-      *asT(i) = init_val;
-      *asIndex(i) = init_loc;
-    }
-  }
-
-  //
-  // Copy ctor.
-  //
-  ReduceMaxLoc(const ReduceMaxLoc& other)
-    : m_myID(other.m_myID),
-      m_blockdata(other.m_blockdata),
-      m_idxdata(other.m_idxdata),
-      m_reduced_val(other.m_reduced_val),
-      m_reduced_idx(other.m_reduced_idx),
-      m_is_copy(true)
-  {
-  }
-
-  //! Destructor releases shared memory block and id
-  ~ReduceMaxLoc()
-  {
-    if (m_is_copy)
-      return;
-    releaseCPUReductionId(m_myID);
-  }
-
-  //! Operator that returns reduced max value.
-  operator T()
-  {
-    int nthreads = omp_get_max_threads();
-    for (int i = 0; i < nthreads; ++i) {
-      Reduce()(m_reduced_val, m_reduced_idx, *asT(i), *asIndex(i));
-    }
-    return m_reduced_val;
-  }
-
-  //
-  // Method that returns reduced max value.
-  //
-  T get() {
-    return operator T();
-  }
-
-  //
-  // Method that returns index corresponding to reduced max value.
-  //
-  Index_type getLoc()
-  {
-    int nthreads = omp_get_max_threads();
-    for (int i = 0; i < nthreads; ++i) {
-      Reduce()(m_reduced_val, m_reduced_idx, *asT(i), *asIndex(i));
-    }
-    return m_reduced_idx;
-  }
-
-  //
-  // Method that updates max and index value for current thread.
-  //
-  const ReduceMaxLoc& maxloc(T val, Index_type idx) const
-  {
-    int tid = omp_get_thread_num();
-    Reduce()(*asT(tid), *asIndex(tid), val, idx);
-    return *this;
-  }
-
-private:
-
-  static constexpr const int s_block_offset =
-      COHERENCE_BLOCK_SIZE / sizeof(CPUReductionBlockDataType);
-  static constexpr const int s_idx_offset = COHERENCE_BLOCK_SIZE / sizeof(Index_type);
-
-  int m_myID;
-  CPUReductionBlockDataType *m_blockdata;
-  Index_type *m_idxdata;
-  T m_reduced_val;
-  Index_type m_reduced_idx;
-  bool m_is_copy;
+  using Base = detail::BaseReduceMinLoc<T, detail::ReduceOMPOrdered>;
```

Comment:
Does clang-format fix this formatting?

Reply:
Fix what about it specifically?  I'm not sure I know what you're asking here.

Reply:
You can ignore this comment. I was going through the PR on my laptop yesterday and it looked weird in a narrower window I had up.